src/actions/configuration/first-time-configuration-action.php

```diff
@@ -58,7 +58,17 @@ public function __construct( Options_Helper $options_helper, Social_Profiles_Hel
 	 * @return object The response object.
 	 */
 	public function set_site_representation( $params ) {
-		$failures = [];
+		$failures   = [];
+		$old_values = $this->get_old_values( self::SITE_REPRESENTATION_FIELDS );
+
+		// This field seems to be not used anymore.
+		$old_values = \array_filter(
+			$old_values,
+			static function( $key ) {
+				return $key !== 'description';
+			},
+			\ARRAY_FILTER_USE_KEY
+		);
```

Comment:
Why not remove it from `SITE_REPRESENTATION_FIELDS`?

It seems to be a remnant from when we had a field for the `blogdescription`? Below there is the if/else to update that. And following what we send to the route in `/packages/js/src/first-time-configuration/first-time-configuration-steps.js`' `updateSiteRepresentation` - it is indeed not there.

So unless I'm missing something, you could remove that instead of filtering?

Reply:
Yeah, I know (hence the comment there), this was just my super-cautious nature kicking in and the will to contain scope creep! 😅
Anyway, I removed the filter and the if/else related to the that field now.